crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -419,7 +425,7 @@ fn handle_own_line_comment_in_clause<'a>(
 ///     # Leading `case (x, y, z)` comment
 ///     case _:
 /// ```
-fn handle_match_comment<'a>(
+fn handle_match_case_comment<'a>(
```

Comment:
Do we need this function? i left it because i hadn't worked with match yet, but that case should be handled by the generic if/while/for/try logic